sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -144,9 +144,10 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
     val (comments, code) = input.split("\n").partition(_.startsWith("--"))
 
     // Runs all the tests on both codegen-only and interpreter modes
-    val codegenConfigSets = Array(CODEGEN_ONLY, NO_CODEGEN).map {
-      case codegenFactoryMode =>
-        Array(SQLConf.CODEGEN_FACTORY_MODE.key -> codegenFactoryMode.toString)
+    val codegenConfigSets = Array(("false", "NO_CODEGEN"), ("true", "CODEGEN_ONLY")).map {
```

Comment:
shall we test all the combinations? e.g. `wholeStage=on, codegen=off`

Reply:
ok

Reply:
will this increase too much test time?

Reply:
I will check the time later, too.